packages/esm-ward-app/src/beds/occupied-bed.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { type Patient } from '@openmrs/esm-framework';
+import { type Bed } from '../types';
+import styles from './occupied-bed.scss';
+import { Tag } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import WardPatientCard from '../ward-patient-card/ward-pateint-card.component';
+
+export interface OccupiedBedProps {
+  patients: Patient[];
+  bed: Bed;
+}
+const OccupiedBed: React.FC<OccupiedBedProps> = ({ patients, bed }) => {
+  return (
+    <div className={styles.occupiedBed}>
+      {patients.map((patient, index: number) => {
+        const last = index === patients.length - 1;
+        return (
+          <div key={patient.uuid + ' ' + index}>
```

Comment:
Shouldn't `patient.uuid` be unique?

Reply:
However you should append a namespace for the key. We don't want random components around the app all having their key equal the patient UUID.


```suggestion
          <div key={`occupied-bed-pt-${patient.uuid}`}>
```